wg-component-standard/README.md

```diff
@@ -33,4 +33,17 @@ Develop a standard foundation (philosophy and libraries) for core Kubernetes com
     - [@kubernetes/wg-component-standard](https://github.com/orgs/kubernetes/teams/wg-component-standard) - Component Standard Discussion
 <!-- BEGIN CUSTOM CONTENT -->
 
+## Issues
+* [Project Board](https://github.com/orgs/kubernetes/projects/26)
+* [k/k Issues / PR's needing Review](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=label%3Awg%2Fcomponent-standard+)
+
+Issues are opened in [k/k](https://github.com/kubernetes/kubernetes) or the appropriate project/component repo with the `wg/component-standard` label.  
+To apply this label to an issue using Prow, leave a comment containing `/wg component-standard` on its own line.  
+
+[k/component-base](https://github.com/kubernetes/component-base) is an automated staging repository managed from the ./staging directory in [k/k](https://github.com/kubernetes/kubernetes). Issues are not currently looked at on this repo.  
```

Comment:
Another idea: we could add a note in the component-base README about opening issues directly in k/k and disable issues in the repo (it's an option in Settings). Disabling issues would remove the "Issues" tab from the repo completely. Given that there haven't been any significant issues (only https://github.com/kubernetes/component-base/issues/2) created in the repo, I don't think this should be a problem...

@stealthybox wdyt?

IIRC we haven't disabled issues on any of the staging repos in the past though. cc @kubernetes/owners

Reply:
This SGTM